### DISCUSSION TITLE
Heic encode from webp (all file format) not working properly

### DISCUSSION THREAD

**Discussion by Risiamu** [ImageMagick/ImageMagick#1981] [General]
### Prerequisites

- [*] I have written a descriptive issue title
- [*] I have verified that I am using the latest version of ImageMagick
- [*] I have searched [open](https://github.com/ImageMagick/ImageMagick/issues) and [closed](https://github.com/ImageMagick/ImageMagick/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description
When conventing to heic, the encoder dosen't actually encode it, but only wrap it.
I find something about the license confliction between libheif libx265 and imagemagick in the pullrequest for heic support. and noticed something as this feature is temp banned on win10.

If the feature is still not avilable, it should be disabled completely instead of wrapping everything...
If it is avilable, then is there anything i missed to enable it?

### Steps to Reproduce
1. find any jpg file, like this one 
![1](https://user-images.githubusercontent.com/30680793/81023599-7b706b80-8ea3-11ea-9bf1-957c4966b758.jpg)
2. magick 1.jpg fromjpg.heic 
 (i packed them up)
[DESKTOP.zip](https://github.com/ImageMagick/ImageMagick/files/4578301/DESKTOP.zip)
identify it:
`fromjpg.heic JPEG 1440x1214 1440x1214+0+0 8-bit sRGB 321280B 0.000u 0:00.000`
3.additional
For image i encoded from ffmpeg, the output for identify:
`ffmpegencoded.heic HEIC 1440x1214 1440x1214+0+0 8-bit YCbCr 0.000u 0:00.001`
### System Configuration

- ImageMagick version:
Version: ImageMagick 7.0.10-3 Q16 x64 2020-03-28 http://www.imagemagick.org
Copyright: Copyright (C) 1999-2018 ImageMagick Studio LLC
License: http://www.imagemagick.org/script/license.php
Visual C++: 180040629
Features: Cipher DPC Modules OpenCL OpenMP(2.0)
Delegates (built-in): bzlib cairo flif freetype gslib heic jng jp2 jpeg lcms lqr lzma openexr pangocairo png ps raw rsvg
 tiff webp xml zlib
- Environment (Operating system, version and so on):
win10
- Additional information:

**Comment by dlemstra** [comment]
Encoding a heic image is not supported on Windows. And when you try to write a `.heic` file it will take the input format as the output format. This would be the same as writing an image called `fromjpg.foo`. The output image that will be created is a `jpeg` file.
